lib/cli/src/commands/create_exe.rs

```diff
@@ -666,6 +659,7 @@ fn run_c_compile(
         .arg("-O2")
         .arg("-c")
         .arg(path_to_c_src)
+        .arg("-I.")
```

Comment:
Please do not include the current directory. The reason for this is that if there are accidentally stray header files in the current directory (from which wasmer gets executed from), cc will prefer the header in the current directory, which then leads to linker errors. This happened to me because I had a stray `static_defs.h` in the current directory.

The solution is to make a separate directory for all the header files in the compilation process.

Reply:
The current directory is the tempfile created one, not the one you execute create-exe in. This temp current directory should include the generated static_defs.h.

Reply:
Your solution relies again on the `set_working_dir` which I really believe we shouldn't be using. So I also agree with Felix here